build/pipelines/release.yml

```diff
@@ -23,17 +31,17 @@ jobs:
   - template: ./templates/build-console-int.yml
     parameters:
       platform: x64
-      additionalBuildArguments: /p:WindowsTerminalReleaseBuild=true
+      additionalBuildArguments: /p:WindowsTerminalOfficialBuild=true;WindowsTerminalBranding=Preview

```

Comment:
I'm assuming we'll be changing `WindowsTerminalBranding=Preview` to `Release` at some point in the _near_ future?

Reply:
This pipeline isn't actually used, this is just for illustrative purposes :)

Reply:
oh well then even better 😄